packages/mui-codemod/README.md

```diff
@@ -91,6 +91,18 @@ A combination of all deprecations.
 npx @mui/codemod@latest deprecations/accordion-props <path>
 ```
 
+#### `divider-props`
+
+```diff
+ <Divider
+-    light
```

Comment:
I think we should replace it with `sx={{ opacity: 0.6 }}` instead of removing it, what do you think?

Reply:
we can do it, but i have few doubts.

- is `light` prop exactly equivalent to `sx={{opacity:0.6}}`. if yes, then we should definetly add it, if not then, wouldn't it be breaking change even after applying codemod. this might confuse users

Questions if we are going to add

- if components already have `sx`, should we merge with existing `sx` props?
- if components already have `sx={{opacity:"any value"}}`, should our codemod override their value or ignore?

Reply:
Good questions @sai6855 🤗 

> is light prop exactly equivalent to sx={{opacity:0.6}}

Not exactly, but close enough. The original alpha channel is `0.12`, and with `light` is `0.08`.
`0.08`/`0.12` = `0.666...` (so we could do `0.7` as well, I don't think the difference is noticeable).
This is the easiest way to do it without adding a mode-aware sx prop. We could do

```tsx
sx={(theme) => { borderColor: theme.vars ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)` : alpha(theme.palette.divider, 0.08)}}
```

But then we would have to import the `alpha` function. I think opacity `0.6` (or `0.7`) does the trick well enough.

> if components already have sx, should we merge with existing sx props?

Yes

> if components already have sx={{opacity:"any value"}}, should our codemod override their value or ignore?

Ignore

Reply:
Got it, added codemods to add sx prop and added migration guide

Reply:
Looks good @sai6855!

I think we missed adding it to this README though:

```suggestion
-    light
+   sx={{ opacity : "0.6" }}
```